enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -36,8 +36,10 @@ type MigrationCache struct {
 	eg       *errgroup.Group
 	quitChan chan struct{}
 
-	maxCopiesPerSec int
-	copyChan        chan *copyData
+	maxCopiesPerSec             int
+	copyChan                    chan *copyData
+	copyChanFullWarningInterval time.Duration
+	numCopiesDropped            *int64
```

Comment:
I am curious why the plain int64 won't work here and a pointer works.

Reply:
https://buildbuddy-corp.slack.com/archives/C01D5GHRJ59/p1663866941156229